doc/news/_preparation_next_release.md

```diff
@@ -148,6 +148,21 @@ you up to date with the multi-language support provided by Elektra.
 
 - Added Reviews for all functions contained in the Elektra Core API _(@lawli3t)_
 - Minor readability improvement in [](/doc/tutorials/highlevel.md) _(Tobias Schubert @qwepoizt)_
+- <<TODO>>
+- <<TODO>>
+- Reformat of enumerations in [INSTALL.md](/doc/INSTALL.md) _(Robert Sowula)_
+- Document names of different components. _(Markus Raab)_
+- Update buildserver documentation _(Robert Sowula)_
+- Reworked [METADATA.ini](/doc/METADATA.ini) _(Markus Raab)_
+- Minor rewording in INSTALL.md _(@a-kraschitzer)_
+- Write notes that `\\` are due to shell recorder, and are not to be copied _(Markus Raab)_
+- Add link to [Go](https://github.com/ElektraInitiative/go-elektra) bindings _(Markus Raab)_
+- Fix order of tutorials _(Markus Raab)_
+- Added API-Reviews for multiple functions in the public API _(Stefan Hanreich)_
+- Minor rewording in [java-kdb.md](/doc/tutorials/java-kdb.md) _(@aaronabebe)_
+- Added a short Visual Studio 2019 tutorial (/doc/tutorials/contributing-windows.md) _(Dominic Jäger)_
+- Added hint regarding WSL filesystem configuration (/doc/tutorials/contributing-windows.md) _(@tucek)_
+- Fixed broken link in yanlr-plugin readme _(@lawli3t)_
```

Comment:
It seems something went wrong here with the rebase. Lots of changes are not from this PR (`doc/news/2021-04-12_0.9.5.md`).

Reply:
Thank you for pointing this out! Should now be fixed.